docs/advanced-usage/access-control/index.md

```diff
@@ -2,3 +2,4 @@
 
 - [Kerberos](kerberos) to access restricted resources
 - [VOMS-proxy](voms-proxy) to authenticate with a VOMS proxy certificate
+- [Private Docker registires](private-docker-registries) to access private images
```

Comment:
`registires` typo

Reply:
`s/private/restricted/g` ? Might be a better word going with "access control" topics...

Reply:
I think private registry is what is used all over the place in documentation and stackoverflow/GitHub issues. If one googles "restricted docker registries" the quality of the results is way lower, therefore the common understanding of the concept by newcomers to the project. Said that, if we still want to do the change I am fine with it.